### PR TITLE
Fixed types for ora v0.3.0

### PR DESCRIPTION
**Pull request by screendriver**
By the way: is it possible to add multiple `Definitions by`? For example the original authors and the contributors?

### CODE REVIEW

**Comment by screendriver** [comment]
By the way: is it possible to add multiple `Definitions by`? For example the original authors and the contributors?

**Comment by zhengbli** [comment]
Yes actually that is encouraged. You just separate the contributor names with comma. 
```
// Definitions by: Basarat Ali Syed <https://github.com/basarat/>, Christian Rackerseder <https://github.com/screendriver/>
```

**Comment by screendriver** [comment]
@zhengbli thank you. Fixed 😎

**Comment by vvakame** [comment]
*ora/index.d.ts*

to author (@basarat). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

**Review by zhengbli — CHANGES_REQUESTED** [review]

File: ora/index.d.ts [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+/// <reference types="node" />
 
-type Color = 'black' | 'red' | 'green' | 'yellow' | 'blue' | 'magenta' | 'cyan' | 'white' | 'gray';
-type Text = string;
-interface Options {
-    text?: Text;
-    spinner?: string | { interval?: number; frames: string[]; }
-    color?: Color;
-    stream?: any;
+declare module "ora" {
```
Comment (screendriver) [inline ora/index.d.ts]: Ok. I removed that. Now it should be ok ☺️

**Comment by vvakame** [comment]
unfortunately, conflict occured. could you resolve it?

**Comment by screendriver** [comment]
Done

**Pull request merged**